tools/web-fuzzing-introspection/app/webapp/models.py

```diff
@@ -63,49 +86,33 @@ def has_introspector(self) -> bool:
         return self.introspector_data != None
 
 
-class Project:
```

Comment:
For future references, it's a lot easier to review if the location of things are not mved around. For example, the `Project` class is moved from line 66 to line 26, but I don't see a reason why?

Reply:
The reason why it is moved up is because the typing for the DBSummary constructor requires Project.
https://github.com/ossf/fuzz-introspector/pull/1600/files#diff-2350a52b5fca0958b98e475be7c02794578f01d9a6748996b9ec6257d3acc28aR62

Reply:
Ah, makes sense. Thanks!